src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberFilter/MemberFilter.jsx

```diff
@@ -40,7 +47,8 @@ class MemberFilter extends React.Component {
    * This method apply the selected filter.
    */
   onClickApply = () => {
-    this.props.onClickApplyFilter(this.state.appliedFilter);
+    this.props.setAppliedFilter(this.state.appliedFilter);
+    window.location.href = `/#${Routes.MEMBER_LIST_VIEW}`;
```

Comment:
This will be handled by Link itself
```suggestion
    window.location.href = `/#${Routes.MEMBER_LIST_VIEW}`;
```

Reply:
Done